samples/bluetooth/peripheral/sample.yaml

```diff
@@ -11,3 +11,7 @@ tests:
     platform_whitelist: nucleo_l4r5zi
     depends_on: arduino_spi arduino_gpio
     extra_args: SHIELD=x_nucleo_idb05a1
+  sample.bluetooth.peripheral.nucleo_wb55rg:
```

Comment:
You don't need to create a new entry since there aren't any special build arguments or dependencies for this board. Just add the nucleo board to the `sample.bluetooth.peripheral` entry above

Reply:
Thanks @MaureenHelm , done